src/libcore/raw.rs

```diff
@@ -30,9 +63,13 @@ pub struct Slice<T> {
 
 impl<T> Copy for Slice<T> {}
 
-/// The representation of a Rust closure
+/// The representation of an old Rust closure
```

Comment:
This type will be deleted soon (i.e. O(next release)), I imagine, and any current users are likely actually incorrect, not just outdated.